src/Main.hs

```diff
@@ -35,6 +35,6 @@ main = do Options f c conf <- execParser opts
           cfg <- maybe (pure defaultConfig) parseConfig conf
           cpg <- flip runReaderT cfg $ do
             cs       <- contracts f
-            (v,w,ts) <- loadSpecified (pack . (f ++) . (':' :) <$> c) cs >>= prepareForTest
+            (v,w,ts) <- loadSpecified (pack <$> c) cs >>= prepareForTest
```

Comment:
why'd we remove this?

Reply:
We are using `isSuffixOf` to deal with contact's name now.